manifests/init.pp

```diff
@@ -55,30 +55,23 @@
 #   Name of the utils package, e.g. 'yum-utils', or 'dnf-utils'.
 #
 # @example Enable management of the default repos for a supported OS:
-#   ```yaml
```

Comment:
I've seen it elsewhere that this can be useful. Maybe it's useful to file a feature request for YAML highlighting for examples.